.gitpod.Dockerfile

```
@@ -1,5 +1,20 @@
-FROM deniswsrosa/couchbase7-5868-gitpod
+FROM couchbase:latest
```

```suggestion
FROM couchbase:7.0.0
```
It would be safer to add the image version for future compatibility issues

---

@nithishr On the version wise, we can keep the latest in similar to other examples that I had merged. Later, we can think as to the specific as needed. Hope this is ok as discussed.

Let me see the other suggestions that you had mentioned.